CMakeLists.txt

```diff
@@ -6,6 +6,9 @@ set(CMAKE_CXX_STANDARD 11)
 set(CMAKE_CXX_STANDARD_REQUIRED ON)
 set(CMAKE_CXX_EXTENSIONS OFF)
 
+SET(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -O2")
```

Comment:
This isn't typical for CMake files. The generally accepted practice is setting CMAKE_BUILD_TYPE to Release (or RelWithDebInfo) from the command line.